e2e/types/types.go

```diff
@@ -76,6 +82,14 @@ type Env struct {
 	C2  Cluster `json:"c2"`
 }
 
+type ApplicationStatus string
+
+// WorkloadStatus holds the status of an application on a specific cluster
+type WorkloadStatus struct {
+	Cluster Cluster
```

Comment:
Using cluster looked like a good idea when we discussed in, but when we report errors we cannot log the list of statuses, since it contains unhelpful like cluster.Kubeconfig. It will be easier to use the list of statuses if we include only the cluster name here:

    [{Cluster: "dr1", Status: "found'}, {"Cluster": "dr2", Status: "partial"]}